Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/Commands/Commands.cs

```diff
@@ -479,9 +474,10 @@ public static Command DeleteBinding(int actionMapIndex, int bindingIndex)
                 InputActionSerializationHelpers.DeleteBinding(binding, actionMap);
                 state.serializedObject.ApplyModifiedProperties();
                 state.m_Analytics.RegisterBindingEdit();
-                if (state.selectedBindingIndex >= bindingIndex)
-                    return SelectPrevBinding(state, actionMap);
-                return state.SelectBinding(state.selectedBindingIndex);
+
+                if (isCut)
```

Comment:
This seems separate from analytics ? 
I'm wondering why its in this PR.

Reply:
Its unrelated, will remove

Reply:
@lyndon-unity seems like the diff is gone now after updating this branch based on develop again will mark resolved